src/test/java/com/bettercloud/vault/VaultTests.java

```diff
@@ -73,17 +73,18 @@ public void testVaultWithEmptyKVEnginePathMap() throws VaultException {
     public void testVaultWithoutKVEnginePathMap() throws VaultException {
         Map<String, String> engineKVMap = new HashMap<>();
         engineKVMap.put("/hello", "2");
-        VaultConfig vaultConfig = new VaultConfig().secretsEnginePathMap(engineKVMap);
+        VaultConfig vaultConfig = new VaultConfig();
```

Comment:
why is `.secretsEnginePathMap(engineKVMap)` removed?

Reply:
Because this test is meant to test what happens when no secretsEnginePath is being used. It was broken from the start of forking the repo :P

Reply:
Even if you set `useSecretsEnginePathMap` to `false`, setting the `.secretsEnginePathMap` in the `VaultConfig` already loads all of the secrets into the map, and the `false` doesn't affect anything if the secrets are already in there.

Reply:
Thanks for explaining :-)